src/request/sign-transaction/SignTransactionApi.js

```diff
@@ -4,26 +4,71 @@
 /* global Nimiq */
 /* global AmountTooSmallError, NetworkMissmatchError */
 /* global SignTransactionWithPassphrase, SignTransactionWithPin */
+/* global TransactionType */
+/* global AddressUtils */
+/* global Utf8Tools */
 
 class SignTransactionApi extends PopupApi { // eslint-disable-line no-unused-vars
     /**
      * @param {TransactionRequest} txRequest
      */
     async onRequest(txRequest) {
-        if (Nimiq.Policy.coinsToSatoshis(txRequest.value) < 1) {
+        // Validation
+        if (!txRequest) throw new Error('Invalid request');
+
+        // These checks throw an exception on failure
+        AddressUtils.isUserFriendlyAddress(txRequest.sender);
```

Comment:
...which is quite strange IMHO

Reply:
There's also `AddressUtils.isValidAddress()` which returns a boolean. But it uses the above internally, and I didn't want to create a huge `if()` statement or three checks, thus I thought I'd just use the underlying error-throwing function.